CONTRIBUTING.md

```diff
@@ -78,30 +78,20 @@ det-deploy local fixture-down
 ### Starting manually
 
 Running the parts of a Determined cluster individually can help speed up
-iteration during development. A minimal cluster consists of four services: a
-[PostgreSQL](https://www.postgresql.org/) database, a Determined master, and 
+iteration during development. A minimal cluster consists of three services: a
+[PostgreSQL](https://www.postgresql.org/) database, a Determined master, and
 a Determined agent.
 
+To start the database (running as a background process), do:
```

Comment:
You dont even need this. I script will start a short lived db while the process is running. I think that is a better use case for users that are contributing.

Reply:
Makes sense -- although I think there's value in also having folks running a long-lived DB for development purposes, so I kept a description of how to do that.